Error handling for scripting.executeScript() (#22279)
mdn/content: files/en-us/mozilla/firefox/releases/107/index.md

### Other

> - The `error` property returned when an error occurs in {{WebExtAPIRef("scripting.executeScript")}} now represents any value the script throws or rejects with, such as an error object, primitive, or undefined {{bug(1740608)}}.

Reviewer: ```suggestion
- The `error` property returned when an error occurs in {{WebExtAPIRef("scripting.executeScript")}} now represents any value the script throws or rejects with, instead of being just an object with a message property {{bug(1740608)}}.
```

No need to call out primitives and undefined here.
